recipes/libxnvme/meta.yaml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Please use the same formatting as other recipes, e.g. remove this line.

Reply:
ack